### PR TITLE
Frequency analysis add cipher init options

### PR DESCRIPTION
**Pull request by Hamms**
Formerly, the FA tool would always look like this:

image

And it will still look that way by default. But you can now specify one of two custom initial modes.

Caesar Cipher:

image

or Substitution Cipher:

image

### CODE REVIEW

**Comment by islemaster** [comment]
:one: Intentional that this PR base isn't staging?

File: dashboard/app/views/levels/editors/_frequency_analysis.html.haml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 .field
+  = f.label :cipher
+  = f.select :cipher, options_for_select(@level.class.ciphers, @level.cipher)
```
Comment (islemaster) [inline dashboard/app/views/levels/editors/_frequency_analysis.html.haml:4]: Worth a little inline documentation here describing the difference between the modes?

Comment (Hamms) [reply dashboard/app/views/levels/editors/_frequency_analysis.html.haml:4]: good call; done

**Comment by Hamms** [comment]
There are no tests touching this feature, which means my next task for this sprint is to write some!

**Pull request merged**